kas-rusty-kaspad/umbrel-app.yml

```
@@ -0,0 +1,31 @@
+manifestVersion: 1
+id: kas-rusty-kaspad
+category: bitcoin
+name: Rusty Kaspad
+version: "v1.0.0"
+tagline: A Rust implementation of a Kaspa full node
+description: |
+  Rusty Kaspad is a Rust implementation of a Kaspa full node. This node software 
+  provides essential Kaspa network services including peer-to-peer networking and 
+  RPC functionality. 
+  
+  Kaspa is a proof-of-work cryptocurrency with instant
+  transaction confirmation through blockDAG technology.
+
+  The very heart of Kaspa is the public nodes. Please consider making your node public
+  by forwarding TCP port 16111 on your firewall to your Umbrel node IP address to support the network.
+
+  This package uses the Docker container maintained by supertypo
+  (https://hub.docker.com/r/kaspanet/rusty-kaspad), based on Rusty Kaspa (https://github.com/kaspanet/rusty-kaspa).
+
+  Packaged for Umbrel by Luke Dunshea (http://luke.dunshea.au).
```

Adapted the description with proper formatting.
```suggestion
  Rusty Kaspad is a Rust implementation of a Kaspa full node. This node software provides essential Kaspa network services including peer-to-peer networking and RPC functionality. 


  Kaspa is a proof-of-work cryptocurrency with instant transaction confirmation through blockDAG technology.


  The very heart of Kaspa is the public nodes. Please consider making your node public by forwarding TCP port 16111 on your firewall to your Umbrel node IP address to support the network.


  This package uses the Docker container maintained by [supertypo](https://hub.docker.com/r/kaspanet/rusty-kaspad), based on [Rusty Kaspa](https://github.com/kaspanet/rusty-kaspa).


  Packaged for Umbrel by Luke Dunshea (http://luke.dunshea.au).
```